sycl/include/CL/sycl/kernel_bundle.hpp

```diff
@@ -158,22 +158,6 @@ class __SYCL_EXPORT kernel_bundle_plain {
   bool native_specialization_constant() const noexcept;
 
 protected:
-  // \returns true if the kernel_bundle has the specialization constant with
-  // specified ID
-  bool has_specialization_constant(unsigned int SpecID) const noexcept;
```

Comment:
Removing spec consts related function to avoid having ABI breakage if it turns out they are not suitable.